### ISSUE TITLE
Add borders to interactive components

### ISSUE THREAD

**Issue by WhiteHoodHacker** [sigpwny/sigpwny.com#123]
This serves the purposes of increasing contrast (e.g. black image on a black background, primary color does not contrast against surface 300) and increasing accessibility (e.g. focus states).

Further reference: https://fluent2.microsoft.design/elevation
